paths_cli/wizard/openmm.py

```diff
@@ -0,0 +1,80 @@
+from paths_cli.wizard.errors import FILE_LOADING_ERROR_MSG, not_installed
+from paths_cli.wizard.core import get_object
+try:
+    from simtk import openmm as mm
+    import mdtraj as md
+except ImportError:
+    HAS_OPENMM = False
+else:
+    HAS_OPENMM = True
+
+
+def _openmm_serialization_helper(wizard, user_input):  # no-cov
+    wizard.say("You can write OpenMM objects like systems and integrators "
+               "to XML files using the XMLSerializer class. Learn more "
+               "here: \n"
+               "http://docs.openmm.org/latest/api-python/generated/"
+               "simtk.openmm.openmm.XmlSerializer.html")
```

Comment:
please add a test that will check page is still reachable (openmm is redesigning part of their website)

Reply:
Added! It would be nice if there was a way to automatically extract URLs, but that's harder than it looks (see for example this URL, which uses Python's implicit string concatenation). In any case, it is probably best practice to keep any URLs like this in variables (as I do now) which keeps help strings readable in the code, as well as facilitating testing.